scripts/ci-entrypoint.sh

```diff
@@ -131,6 +131,59 @@ create_cluster() {
     "${REPO_ROOT}/hack/create-dev-cluster.sh"
 }
 
+install_addons() {
+    # Copy the kubeadm configmap to the calico-system namespace. This is a workaround needed for the calico-node-windows daemonset to be able to run in the calico-system namespace.
+    "${KUBECTL}" create ns calico-system
+    "${KUBECTL}" get configmap kubeadm-config --namespace=kube-system -o yaml \
+    | sed 's/namespace: kube-system/namespace: calico-system/' \
+    | ${KUBECTL} create -f -
+
+    # install cni
+    echo "Installing Calico CNI via helm"
+    "${HELM}" repo add projectcalico https://projectcalico.docs.tigera.io/charts
+    "${HELM}" install calico projectcalico/tigera-operator -f "${CALICO_VALUES:-${REPO_ROOT}/templates/addons/calico/values.yaml}" --namespace tigera-operator --create-namespace
+
+    export -f wait_for_nodes
+    timeout --foreground 1800 bash -c wait_for_nodes
+
+    # Add FeatureOverride for ChecksumOffloadBroken in FelixConfiguration.
+    # This is the recommended workaround for https://github.com/projectcalico/calico/issues/3145.
+    "${KUBECTL}" apply -f "${REPO_ROOT}"/templates/addons/calico/felix-override.yaml
```

Comment:
Do we need to do the Windows foo here as well for Windows scenarios that use ci-entrypoint?